content/en/docs/concepts/architecture/cloud-controller.md

```diff
@@ -43,11 +43,11 @@ The controllers inside the cloud controller manager include:
 
 ### Node controller
 
-The node controller is responsible for creating {{< glossary_tooltip text="Node" term_id="node" >}} objects
+The node controller is responsible for updating {{< glossary_tooltip text="Node" term_id="node" >}} objects
 when new servers are created in your cloud infrastructure. The node controller obtains information about the
 hosts running inside your tenancy with the cloud provider. The node controller performs the following functions:
 
-1. Initialize a Node object for each server that the controller discovers through the cloud provider API.
+1. Update a Node object with its server's unique identifier obtained from the cloud provider API.
```

Comment:
This sentence feels a bit clunky. A non-blocking nit, however, would rephrasing it to "Update a Node object with the corresponding server's unique identifier obtained from the cloud provider API." work?

Reply:
sure thing, i don't mind making that adjustment